1. Случайная непрерывная величина A имеет равномерное распределение на промежутке (200, 800].
Найдите ее среднее значение и дисперсию.


$M(X) = \dfrac{a+b}{2}$

$D(X) = \dfrac{(b-a)^2}{12}$

In [1]:
a = 200 
b = 800
M = (a + b) / 2
M

500.0

In [2]:
D = ((b - a) ** 2) / 12
D

30000.0

#### Ответ:

Среднее значение M = 500

Дисперсия D = 30000

2. О случайной непрерывной равномерно распределенной величине B известно, что ее дисперсия равна 0.2.
Можно ли найти правую границу величины B и ее среднее значение зная, что левая граница равна 0.5?
Если да, найдите ее.


Вроде бы найти эти значения можно, т.к. они легко выводятся из формул дисперсии и математического ожидания для равномерно распределенной величины.

In [3]:
a = 0.5
D = 0.2

$D(X) = \dfrac{(b-a)^2}{12}$

$12D = b^2 - 2ab + a^2$

$b^2 - b - 2.15 = 0$

В результате решения квадратного уравнения получается 2 корня (-1.05 и 2.05), один из которых отрицательный и не подходит на роль правой границы, т.к. правая граница должна быть больше левой.

In [4]:
b = 2.05

$M(X) = \dfrac{a+b}{2}$

In [5]:
M = (a + b) / 2
M

1.275

#### Ответ:

Правая граница b = 2.05
Среднее значение M = 1.275

3. Непрерывная случайная величина X распределена нормально и задана плотностью распределения


f(x) = (1 / (4 * sqrt(2*pi))) * (exp(-(x+2) * *2) / 32).

Найдите:
а). M(X)
б). D(X)
в). std(X) (среднее квадратичное отклонение)


Сопоставив заданную функцию плотности с функцией плотности нормального распределения по определению, несложно увидеть чему будут равны искомые значения.

$f(x) = \dfrac{1}{\sqrt{2\pi \sigma^2}} \cdot e^{-\dfrac {(x-\mu)^2}{2\sigma^2}}$

$f(x) = \dfrac{1}{4\sqrt{2\pi}} \cdot e^{-\dfrac {(x+2)^2}{32}}$

#### Ответ:

$D(X) = 16$

$M(X) = \mu = -2$

$\sigma = \sqrt{D(X)} = 4$


4. Рост взрослого населения города X имеет нормальное распределение.
Причем, средний рост равен 174 см, а среднее квадратичное отклонение равно 8 см.
Какова вероятность того, что случайным образом выбранный взрослый человек имеет рост:

а). больше 182 см

б). больше 190 см

в). от 166 см до 190 см

г). от 166 см до 182 см

д). от 158 см до 190 см

е). не выше 150 см или не ниже 190 см

ё). не выше 150 см или не ниже 198 см

ж). ниже 166 см.


$\mu = 174$

$\sigma = 8$

In [6]:
mu = 174
std = 8

In [7]:
def find_z(x):
    return (x - mu) / std

In [8]:
z1 = find_z(182)
z1

1.0

In [10]:
p1 = 0.16

In [11]:
z2 = find_z(190)
z2

2.0

In [12]:
p2 = 0.023

In [15]:
z3a = find_z(166)
z3a

-1.0

In [14]:
p3 = 0.818

In [16]:
p4 = 0.681

In [18]:
z5a = find_z(158)
z5a

-2.0

In [19]:
p5 = 0.954

In [21]:
z6a = find_z(150)
z6a

-3.0

In [22]:
p6 = 0.025

In [25]:
z7b = find_z(198)
z7b

3.0

In [23]:
p7 = 0.004

In [27]:
z8 = find_z(166)
z8

-1.0

In [28]:
p8 = 0.159

#### Ответ:

а) Больше 182см = 0.16

б) Больше 190см = 0.023

в) От 166см до 190см = 0.818

г) От 166см до 182см = 0.681

д) От 158см до 190см = 0.954

е) Не выше 150см или не ниже 190см = 0.025

ё) Не выше 150см или не ниже 198см = 0.004

ж) Ниже 166см = 0.159


5. На сколько сигм (средних квадратичных отклонений) отклоняется рост человека, равный 190 см, от
математического ожидания роста в популяции, в которой M(X) = 178 см и D(X) = 25 кв.см?


In [30]:
from math import sqrt
x = 190
M = 178
D = 25

In [31]:
std = (x - M) / sqrt(D)
std

2.4

#### Ответ:

отклоняется на 2.4 сигмы.